tests/testmodels.py

```diff
@@ -150,6 +151,9 @@ class ModelTestPydanticMetaBackwardRelations3(Model):
 class Node(Model):
     name = fields.CharField(max_length=10)
 
+    class Meta:
+        indexes = [Index(fields=("name",))]
```

Comment:
It looks like you added this index because it exposes the issue with serializing `Index` in `Model.describe`. I was able to identify it only because I worked on the same issue in my PR, it wouldn't be clear to anyone reviewing this PR.

Even though this index exposes an issue, it isn't documented and completely not obvious why it is here. Someone can remove it later without knowing its purpose and the same regression can happen again. I added a dedicated class for testing indexes in my PR.